base/dcp_feed_type.go

```diff
@@ -171,6 +174,10 @@ func cbgtFeedParams(spec BucketSpec, scope string, collections []string, dbName
 		feedParams.Collections = collections
 	}
 
+	if spec.Serverless == true {
+		feedParams.DCPBuffer = 1000000
```

Comment:
This is hardcoded, but we have a value on spec we could be using?

Reply:
As far as I am aware they are in different units for some reason so I was setting manually to 1000000 bytes whereas the gocb params take 1 * 1024 * 1024 for 1MB.